contracts/price/ZNSCurvePricer.sol

```diff
@@ -61,6 +61,9 @@ contract ZNSCurvePricer is AAccessControlled, ARegistryWired, UUPSUpgradeable, I
         bytes32 parentHash,
         string calldata label
     ) public view override returns (uint256) {
+        // Confirms string values are only [a-z0-9]
+        label.validate();
```

Comment:
why do we do it in this function as well? we will do this twice for every register